src/11_args.py

```diff
@@ -44,6 +53,10 @@
 # Note: Google "python keyword arguments".
 
 # YOUR CODE HERE
+def f4(*args ,**kwargs):
+	dictionary = kwargs if kwargs else args[0]
+	for key, value in dictionary.items():
+		print(f"key: {key}, value: {value}")
```

Comment:
This is a great solution! Way to make it dynamic! You'll find this pattern comes in handy with things like closures and creating dynamic higher-order/wrapper functions.